lib/ae_mdw/names.ex

```diff
@@ -175,8 +182,58 @@ defmodule AeMdw.Names do
     end
   end
 
-  defp build_height_streamer(%{owned_by: _owner_pk}, _state, _scope, _cursor) do
-    raise(ErrInput.Query, value: "can't order by expiration when filtering by owner")
+  defp build_height_streamer(%{owned_by: owner_pk, state: "active"}, state, scope, cursor) do
+    scope = serialize_owner_expiration_scope(owner_pk, scope)
+    cursor = serialize_owner_expiration_cursor(owner_pk, cursor)
+
+    fn direction ->
+      state
+      |> Collection.stream(@table_active_owner_expiration, direction, scope, cursor)
+      |> Stream.map(fn {^owner_pk, expiration_height, plain_name} ->
+        {{expiration_height, plain_name}, :active}
+      end)
+    end
+  end
+
+  defp build_height_streamer(%{owned_by: owner_pk, state: "inactive"}, state, scope, cursor) do
+    scope = serialize_owner_expiration_scope(owner_pk, scope)
```

Comment:
nit pick for no sending through the wires so instead of `serialize` could it be a `get`? (also because all over the places we are using serialize when encoding the cursor so serialize cursor on the line below gets mixed with the same naming for other purpose)

Reply:
As the scope concept is already been applied to `gen:first_gen-last_gen` and `txi:first_txi-last_txi` may we continue calling it `key_boundary` (or other better name if you want)? I understand you're avoiding big names (: but I think it's needed because it's not a scope.

Reply:
Done, renamed the variables/functions